System.IO.Ports/SerialDeviceController.cs

```diff
@@ -16,7 +16,7 @@ internal static class SerialDeviceController
         private static object _syncLock;
 
         // backing field for DeviceCollection
-        private static ArrayList s_deviceCollection;
+        private static ArrayList _deviceCollection;
```

Comment:
you will have to rename native and adjust the native signature. Not really sure it's really valuable?